common/common-protobuf/src/main/java/org/apache/servicecomb/codec/protobuf/definition/ProtobufManager.java

```diff
@@ -37,6 +39,9 @@
 
   private static final Object LOCK = new Object();
 
+  private static ScopedProtobufSchemaManager defaultScopedProtobufSchemaManager = new ScopedProtobufSchemaManager(
```

Comment:
It could cause some trouble as the schema manager's class loader cannot be changed after the static initial time.

Reply:
this is for app classloader, will never changed.